modules/ROOT/pages/apikit-4-soap-fault-task.adoc

```diff
@@ -3,12 +3,12 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can use SoapUI to identify an operation you need to implement by making the API fail. You use the `OrderTshirt` operation to create a request in SOAP UI to place an order for a large t-shirt for John. The API cannot handle this operation and a SOAP fault occurs.
+Use SoapUI to identify an operation you implement by making the API fail. Create a request in SoapUI using the `OrderTshirt` operation to place an order for a large t-shirt for John. As the API cannot handle this operation, a SOAP fault occurs.
```

Comment:
I find this paragraph confusing. 

- Is "SoapUI" a term or should you say "Use the Soap UI...)?
- What is the goal? Is it to test what happens when an operation in the API fails or test an operation that only occurs when the API fails?
- Is OrderTshirt an example of an operation? If so, you need to introduce example - "For example...suppose you create a request in SoapUI using the...
- "As" > "Because" ("as" implies time, so it should not be used as a substitute for "because")

Reply:
SoupUI is software for making SOAP requests. In previous articles, the readers were requested to download the software.